__tests__/app.test.js

```diff
@@ -180,4 +180,72 @@ describe("app", () => {
         .expect(404)
     });
   });
-}); 
\ No newline at end of file
+}); 
+
+describe("GET /api/articles/:article_id/comments", () => {
+  test("Responds with a status of 200 for the right request.", () => {
+    return request(app).get("/api/articles/1/comments").expect(200);
+  });
+  test("200 - Responds with the requested article_id to the client.", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body }) => {
+        const comments = body.article;
+        expect(Object.keys(comments[0]).length).toBe(6);
+        expect(comments).toHaveLength(11);
```

Comment:
Great that you've checked the length of the array to avoid a possible false positive test.